pom.xml

```diff
@@ -93,7 +93,7 @@
     <metrics.client.version>0.4.3</metrics.client.version>
     <mockito.version>1.10.19</mockito.version>
     <slf4j.version>1.7.12</slf4j.version>
-    <vertx.version>2.1.6</vertx.version>
+    <vertx.version>3.2.1</vertx.version>
```

Comment:
Can you add a comment to the README about using version 1.0 with vertx 2.x and that we'll only support vertx 3 going forward?